packaging/generic-deb/debian/README.source

```diff
@@ -0,0 +1,14 @@
+Steps to build and install package:
+
+1. Install build dependencies
+    sudo apt-get update -y && sudo apt-get install -y devscripts build-essential lintian git curl golang-go debhelper
+
+2. Build the package by running
+    make clean deb
+
+3. Install the package with
+    sudo dpkg -i amazon-ecs-init_1.45.0-1_amd64.deb
+
+4. Install docker and start ecs service
+    sudo apt-get install docker.io -y # or via docker repos: https://docs.docker.com/engine/install/
```

Comment:
The ecs-init package depends on a docker installation, so this should happen first. Although if you use the `apt` invocation I described above to install the ecs-init package, Docker will be installed automatically.

It's probably fine to list the docker installation as an explicit step, though, in case the user wants to customize the configuration or something like that before setting up ECS. Just put it before the ecs-init installation in the steps.

Reply:
building ecs-init doesn't require docker, so I don't think it's necessary to install before building the deb. We download the agent container from s3 so we are not building any docker containers as part of this packaging

Reply:
My comment was with respect to the installation of ecs-init, not building it. Build happens in step 1. You have ecs-init installation happening in step 3, and docker installation in step 4. You either need to re-order 3 and 4, or combine them.

Reply:
done